index.html

```diff
@@ -0,0 +1,118 @@
+                        </div>
+                    </div>
+               
+
+                <form>
+                <div class="element-style">
+                    <div>
+                        <label>Email address</label>
+                        <input type="email" name="email">
+                    </div>
+
+                    <div>
+                        <label>Phone (recommended)</label>
+                        <input type="tel" name="tel">
+                        <p>Standard call, messaging or data rates may apply.</p>
+                    </div>
+                    
+                    <div>
+                        <label>Password</label>
+                        <input type="password" name="password">
+                    </div>
+                    
+                   <div>
+                    <button type=button><i class="material-icons">lock</i> <span>Create Account</span></button>
+                   </div>
+                    
+                </div>
+                  <div class="click-info">
+                    <p><span class="spacing">By clicking Create account, you agree to our <a href="https://www.mint.com/terms" rel="noopener noreferrer" target="_blank"><span class="blue">Terms</span></a> </span>and have read and acknowledge our <a href="https://www.intuit.com/privacy/" target="_blank" rel="noopener noreferrer"> <span class="blue">Privacy </span></a> <a href="https://www.intuit.com/privacy/" target="_blank" rel="noopener noreferrer"><span class="center blue">Statement.</span></a> </p>
+                  </div>  
+                  
+                  <div class="back-color">
+                  <div class="captcha">
+                      <p>Invisible reCAPTCHA by Google <a href="https://policies.google.com/privacy?hl=en" target="_blank" rel="noopener noreferrer"><span>Privacy Policy</span> </a> and <a href="https://policies.google.com/terms?hl=en" target="_blank" rel="noopener noreferrer"><span>Terms of Use.</span> </a></p>
+                </div>
+            </div>
+        
+        </form>
+    </div>
+        </section>
+        
+        
+        <footer>
+            <div class="policies">
+                <p><a href="https://www.intuit.com/legal/" target="_blank" rel="noopener noreferrer">Legal</a> <a href="https://policies.google.com/privacy?hl=en" rel="noopener noreferrer" target="_blank">Privacy</a> <a href="http://security.intuit.com/" rel="noopener noreferrer" target="_blank">Security</a></p>
+            </div>
+            <div class="line1">
+                <p>© 2020 Intuit, Inc. All rights reserved. Intuit, QuickBooks, QB, TurboTax, ProConnect and Mint are registered trademarks of Intuit Inc.</p>
+            </div>
+            <div class="line2">
+                <p>Terms and conditions, features, support, pricing, and service options subject to change without notice.</p>
+            </div>
```

Comment:
[OPTIONAL] Its usually great to enclose the links with `nav` tag.

[OPTIONAL] Add padding between the links and also change their color to match the original page.